rdbi-recipes/src/main/java/com/lithium/dbi/rdbi/recipes/channel/ChannelPublisher.java

```diff
@@ -13,7 +13,7 @@
 public class ChannelPublisher {
 
     private static final int DEFAULT_CHANNEL_DEPTH = 100;
-    private static final int DEFAULT_CHANNEL_EXPIRE_SECONDS = 60 * 60; // 1 hour
+    private static final int DEFAULT_CHANNEL_EXPIRE_SECONDS = 30 * 24 * 60 * 60; // 30 days
```

Comment:
would prefer Duration.ofDays(30).toStandardSeconds()

Reply:
that requires a cast from long -> int. Still yes?

Reply:
yea, i'd say so. not particularly worried about int overflow w/ secs on any duration we'd reasonably specify here.